packages/drivers/odsp-driver/src/getFileLink.ts

```diff
@@ -101,12 +100,20 @@ async function getFileLinkCore(
             let additionalProps;
             const fileLink = await getWithRetryForTokenRefresh(async (options) => {
                 attempts++;
-                const token = await getToken({ ...options, ... odspUrlParts });
+                const storageTokenFetcher = toInstrumentedOdspTokenFetcher(
+                    logger,
+                    odspUrlParts,
+                    getToken,
+                    true /* throwOnNullToken */,
+                );
+                const storageToken = await storageTokenFetcher(options,"GetSharingLinkTokenCore");
```

Comment:
This will have `_GetToken` appended at the end so let's call it to match the function name
```suggestion
                const storageToken = await storageTokenFetcher(options,"GetFileLinkCore");
```